### ISSUE TITLE
Can't move note horizontally without it resnapping

### ISSUE THREAD

**Issue by YaLTeR** [Quaver/Quaver#1878]
If the beat snap that the note is on is not currently active (e.g. the note is on 1/3 and you're on 1/4) then it's impossible to move the note across lanes without it resnapping.

**Comment by Swan** [comment]
This is the intended behavior.

**Comment by YaLTeR** [comment]
I'm not sure it's good intended behavior. I had a track with an 1/4 snap and a few singular notes on 1/12, so I was generally working in the 1/4 snap mode. I was doing some column swapping on the notes and I had to change to 1/12 and back for every one of those few 1/12 notes that I had to move.

Why not have a few vertical pixels where if you drag within those pixels (don't move the mouse vertically basically) the note remains in its position instead of re-snapping, and if you move slightly further vertically then it resnaps?
